CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Added `miden_core::utils::sync::racy_lock` module (#1463).
 - Updated `miden_core::utils` to re-export `std::sync::LazyLock` and `racy_lock::RacyLock as LazyLock` for std and no_std environments, respectively (#1463).
 - Made the undocumented behavior of the VM with regard to undefined behavior of u32 operations, stricter (#1480)
+- Debug instructions can be enabled in the cli `run` command using `--debug` flag
```

Comment:
We usually add PR numbers (in parenthesis at the end of the line).

Also, I would add a line about changing the constructor of the `ExecutionOptions` struct and mark it as `[BREAKING]`.